great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -55,6 +55,10 @@ class ParameterBuilder(Builder, ABC):
         ```
     """
 
+    exclude_field_names: Set[str] = {
+        "data_context",
+    }
+
```

Comment:
How is this leveraged? I don't love explicitly noting field names since this seems hard to maintain but I'm not sure of a great alternative.

Reply:
@cdkini We have this pattern -- in a few places (including the Stores/Schemas).  Happy to describe how it works on a call.  Thanks.